src/easylink/pipeline_schema_constants/development.py

```diff
@@ -208,6 +208,14 @@
             "complex": {
                 "step": HierarchicalStep(
                     step_name="step_5_and_6",
+                    input_slots=[
```

Comment:
This step wasn't properly formed in the development schema and was caught by the new _extract_output_slot test